src/utilities/color-manipulation.ts

```diff
@@ -8,7 +9,7 @@ export function lightenColor(color: HSLColor | string, lighten = 0) {
   const {lightness} = color;
   const nextLightness = lightness + lighten;
 
-  return {...color, lightness: nextLightness};
+  return {...color, lightness: clamp(nextLightness, 0, 100)};
```

Comment:
This fixes a bug where invalid hex colors could be returned if a lightness was outside the valid range.

It could use a regression test.